src/edu/csus/ecs/pc2/clics/API202306/CLICSAccount.java

```diff
@@ -0,0 +1,320 @@
+        Account [] newaccounts = null;
+        Log log = StaticLog.getLog();
+
+        try {
+            ObjectMapper mapper = new ObjectMapper(new YAMLFactory());
+            newaccounts = createAccountsFromCLICSAccounts(contest, mapper.readValue(yamlfile, CLICSAccount[].class), site, log);
+        } catch (Exception e) {
+            log.log(Log.WARNING, "could not deserialize account file " + yamlfile, e);
+        }
+        return(newaccounts);
+     }
+
+    /**
+     * Create account list from a string containing accounts yaml
+     *
+     * @param contest the contest (needed for groups)
+     * @param yaml yaml to deserialize
+     * @param site the site to create the accounts for
+     * @return array of accounts to add, or null on error
+     */
+    public static Account [] fromJSON(IInternalContest contest, String json, int site) {
+        Account [] newaccounts = null;
+        Log log = StaticLog.getLog();
+
+        try {
+            ObjectMapper mapper = new ObjectMapper();
+            newaccounts = createAccountsFromCLICSAccounts(contest, mapper.readValue(json, CLICSAccount[].class), site, log);
+        } catch (Exception e) {
+            log.log(Log.WARNING, "could not deserialize team string", e);
+        }
+        return(newaccounts);
+     }
+
+    /**
+     * Create account list from a string containing accounts yaml
+     *
+     * @param contest the contest (needed for groups)
+     * @param yaml yaml to deserialize
+     * @param site the site to create the accounts for
+     * @return array of accounts to add, or null on error
+     */
+    public static Account [] fromYAML(IInternalContest contest, String yaml, int site) {
+        Account [] newaccounts = null;
+        Log log = StaticLog.getLog();
+
+        try {
+            ObjectMapper mapper = new ObjectMapper(new YAMLFactory());
+            newaccounts = createAccountsFromCLICSAccounts(contest, mapper.readValue(yaml, CLICSAccount[].class), site, log);
+        } catch (Exception e) {
+            log.log(Log.WARNING, "could not deserialize team string", e);
+        }
+        return(newaccounts);
+     }
+
+    /**
+     * Converts CLICS teams into a PC2 Account array.
+     *
+     * @param contest The contest
+     * @param accts array of CLICS accounts
+     * @param site The site to create accounts on
+     * @param log For errors
+     * @return an Account array of accounts converted from the CLICS accounts or null if error
+     */
+    private static Account [] createAccountsFromCLICSAccounts(IInternalContest contest, CLICSAccount [] accts, int site, Log log) {
+
+        Account [] newaccounts = null;
+        ArrayList<Account> accounts = new ArrayList<Account>();
+        Account account;
+        JSONTool jsontool = new JSONTool(contest, null);
+        PermissionList teamPermissionList = new PermissionGroup().getPermissionList (ClientType.Type.TEAM);
```

Comment:
It does not appear that either of the above two declared variables are ever used...

Reply:
Kind of inherited.  Basically, I copied Doug's (probably) original code in the comment to use as  a model of what had to be done, and then went through and just took what I needed.  I have removed those variables as well as the comment block of old code.  Fixed in next push.